I'm sorry, but an uncaught exception occurred.

After initialization, but before game start.
  File "renpy/common/00start.rpy", line 83, in _init_language
    renpy.change_language(language)
  File "game/screens.rpy", line 1694, in <module>
    background Frame((255, 255, 255, 150))
Exception: Not a displayable: (255, 255, 255, 150)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/bootstrap.py", line 275, in bootstrap
    renpy.main.main()
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/main.py", line 670, in main
    run(restart)
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/main.py", line 74, in run
    renpy.translation.init_translation()
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/translation/__init__.py", line 593, in init_translation
    renpy.store._init_language() # type: ignore
  File "renpy/common/00start.rpy", line 83, in _init_language
    renpy.change_language(language)
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/translation/__init__.py", line 685, in change_language
    new_change_language(tl, language)
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/translation/__init__.py", line 639, in new_change_language
    i.apply()
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/ast.py", line 2974, in apply
    value = renpy.python.py_eval(expr)
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screens.rpy", line 1694, in <module>
    background Frame((255, 255, 255, 150))
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/display/imagelike.py", line 210, in __init__
    self.image = renpy.easy.displayable(image)
  File "/Users/oliviamei/Downloads/renpy-8.1.3-sdk/renpy/easy.py", line 145, in displayable
    raise Exception("Not a displayable: %r" % (d,))
Exception: Not a displayable: (255, 255, 255, 150)

macOS-14.4.1-arm64-arm-64bit arm64
Ren'Py 8.1.3.23091805
dreamscapes 1.0
Mon Apr  1 01:52:22 2024
